README.md

```diff
@@ -10,32 +10,33 @@ This repository contains the backend logic for the dfusion exchange based on the
   <img src="documentation/architecture.png" alt="dex-services architecture" width="500">
 </p>
 
-The *Event Listener* registers for certain EVM events via the [Gnosis Trading DB](https://github.com/gnosis/pm-trading-db).
+The _Event Listener_ registers for certain EVM events via the [Gnosis Trading DB](https://github.com/gnosis/pm-trading-db).
```

Comment:
Do we really want these changes? Would you mind turning off autoformat on save for the README file?